apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx

```diff
@@ -264,7 +259,7 @@ export default function ReceiveConfirmation({ navigation, route }: Props) {
         {t("transfer.receive.shareAddress")}
       </Button>}
       </Flex>
-     {verified ? null : <ReceiveSecurityModal onVerifyAddress={onRetry} />}
+     {verified ? null : isModalOpened ? <ReceiveSecurityModal onVerifyAddress={onRetry} /> : null}
```

Comment:
Maybe it could be written like this : `(isModalOpened && !verified) ? <ReceiveSecurityModal onVerifyAddress={onRetry} /> : null` ?

Feel free to resolve if it's less readable